src/rf/apps/workers/process.py

```diff
@@ -228,6 +228,7 @@ def thumbnail(self, record):
                                                enums.STATUS_THUMBNAIL)
 
         if make_thumbs_for_layer(layer_id):
+            log.info('Done generating thumbnail')
```

Comment:
If we have 1 worker per instance in an auto scaling group, then each one should be writing to its own file.